src/Bicep.LangServer/Snippets/Templates/res-logic-app-from-file.bicep

```diff
@@ -0,0 +1,8 @@
+// Logic App
+resource /*${1:logicApp}*/logicApp 'Microsoft.Logic/workflows@2019-05-01' = {
+  name: /*${2:'name'}*/'name'
+  location: resourceGroup().location
+  properties: {
+    definition: json(loadTextContent(/*${3:'REQUIRED'}*/'REQUIRED')).definition

```

Comment:
Instead required put logicApp.json it has to be nam of the file with definition. In addition, remove the `.definition` from the end and from json file just leave object starting from schema.

Reply:
The thing is that there are 2 tests. One that uses mapping done in main.bicep and second that just compiles the snippet. Your code is failing with the second one as there is no REQUIRED file to be loaded. 
@anthony-c-martin should we make an exemption from the REQUIRED rule and allow for parameter that needs to be changed different file name?